strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/config/Maven2LayoutProviderConfig.java

```diff
@@ -1,100 +1,107 @@
+		"org.carlspring.strongbox.storage" })
 @Import(MavenIndexerConfig.class)
-public class Maven2LayoutProviderConfig
-{
-
-    public static final String FILE_SYSTEM_ALIAS = "LayoutFileSystemFactory." + Maven2LayoutProvider.ALIAS;
-
-    public static final String FILE_SYSTEM_PROVIDER_ALIAS = "LayoutFileSystemProviderFactory." +
-                                                            Maven2LayoutProvider.ALIAS;
-
-    @Inject
-    protected StorageProviderRegistry storageProviderRegistry;
-
-    @Inject
-    private Environment environment;
-
-
-    @Bean(FILE_SYSTEM_PROVIDER_ALIAS)
-    public LayoutFileSystemProviderFactory mavenRepositoryFileSystemProviderFactory()
-    {
-        return (repository) -> {
-            StorageProvider storageProvider = storageProviderRegistry.getProvider(repository.getImplementation());
-
-            LayoutFileSystemProvider result;
-            if (Boolean.parseBoolean(environment.getProperty(MavenIndexerEnabledCondition.MAVEN_INDEXER_ENABLED)))
-            {
-                result = indexedMavenFileSystemProvider(storageProvider.getFileSystemProvider());
-            }
-            else
-            {
-                result = mavenFileSystemProvider(storageProvider.getFileSystemProvider());
-            }
-
-            return result;
-        };
-
-    }
-
-    @Bean
-    @Scope("prototype")
-    public Maven2FileSystemProvider mavenFileSystemProvider(FileSystemProvider provider)
-    {
-        return new Maven2FileSystemProvider(provider);
-    }
-
-    @Bean
-    @Scope("prototype")
-    public LayoutFileSystemProvider indexedMavenFileSystemProvider(FileSystemProvider provider)
-    {
-        return new IndexedMaven2FileSystemProvider(provider);
-    }
-
-    @Bean(FILE_SYSTEM_ALIAS)
-    public LayoutFileSystemFactory mavenRepositoryFileSystemFactory(PropertiesBooter propertiesBooter)
-    {
-        LayoutFileSystemProviderFactory providerFactory = mavenRepositoryFileSystemProviderFactory();
-        
-        return (repository) -> {
-            StorageProvider storageProvider = storageProviderRegistry.getProvider(repository.getImplementation());
-
-            return mavenRepositoryFileSystem(propertiesBooter,
-                                             repository,
-                                             storageProvider.getFileSystem(),
-                                             providerFactory.create(repository));
-        };
-    }
-
-    @Bean
-    @Scope("prototype")
-    public MavenFileSystem mavenRepositoryFileSystem(PropertiesBooter propertiesBooter,
-                                                     Repository repository,
-                                                     FileSystem storageFileSystem,
-                                                     LayoutFileSystemProvider provider)
-    {
-        return new MavenFileSystem(propertiesBooter, repository, storageFileSystem, provider);
-    }
+public class Maven2LayoutProviderConfig {
```

Comment:
Please, preserve the original formatting of all files and only make changes to the lines that really need them.

Reply:
Here are some more details on our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html).